src/Tickets/Emails/Emails_Tab.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * Handles registering and setup for the Tickets Emails settings tab.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+use TEC\Tickets\Commerce\Shortcodes\Checkout_Shortcode;
+use TEC\Tickets\Commerce\Shortcodes\Success_Shortcode;
+use TEC\Tickets\Commerce\Gateways\Manager;
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_Gateway as Gateway;
+use TEC\Tickets\Settings as Tickets_Commerce_Settings;
+use Tribe\Tickets\Admin\Settings as Plugin_Settings;
+use \tad_DI52_ServiceProvider;
+use \Tribe__Template;
+use Tribe__Tickets__Main;
+
+/**
+ * Class Emails_Tab
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ */
+class Emails_Tab extends tad_DI52_ServiceProvider {
```

Comment:
I don't think we need this as a service provider. It can be a plain class.

Reply:
You are correct. Thanks!